edge-functions/sveltekit-redirect/svelte.config.js

```diff
@@ -0,0 +1,19 @@
+import adapter from '@sveltejs/adapter-auto';
+import preprocess from 'svelte-preprocess';
+
+/** @type {import('@sveltejs/kit').Config} */
+const config = {
+	// Consult https://github.com/sveltejs/svelte-preprocess
+	// for more information about preprocessors
+	preprocess: [
+		preprocess({
+			postcss: true
+		})
+	],
+
+	kit: {
+		adapter: adapter()
```

Comment:
I think we probably need to use `adapter-vercel` instead of `adapter-auto`, so that we can set `edge: true`. For now we'd also need to specify `ENABLE_VC_BUILD=1`, no?